astropy/modeling/rotations.py

```diff
@@ -27,70 +27,114 @@
 from .parameters import Parameter
 from astropy.coordinates.matrix_utilities import rotation_matrix, matrix_product
 from astropy import units as u
-from astropy.utils.decorators import deprecated
 from .utils import _to_radian, _to_orig_unit
 
 __all__ = ['RotateCelestial2Native', 'RotateNative2Celestial', 'Rotation2D',
-           'EulerAngleRotation']
+           'EulerAngleRotation', 'Rotation3D']
 
 
-class _EulerRotation:
-    """
-    Base class which does the actual computation.
+def _create_matrix(angles, axes_order):
+    matrices = []
+    for angle, axis in zip(angles, axes_order):
+        if isinstance(angle, u.Quantity):
+            angle = angle.value
+        angle = angle.item()
+        matrices.append(rotation_matrix(angle, axis, unit=u.rad))
+    result = matrix_product(*matrices[::-1])
+    return result
+
+
+def spherical2cartesian(alpha, delta):
+    alpha = np.deg2rad(alpha)
+    delta = np.deg2rad(delta)
+    x = np.cos(alpha) * np.cos(delta)
+    y = np.cos(delta) * np.sin(alpha)
+    z = np.sin(delta)
+    return np.array([x, y, z])
+
+
+def cartesian2spherical(x, y, z):
+    h = np.hypot(x, y)
+    alpha = np.rad2deg(np.arctan2(y, x))
+    delta = np.rad2deg(np.arctan2(z, h))
+    return alpha, delta
+
+
+class Rotation3D(Model):
     """
+    Perform a series of rotations about different axis in 3D space.
+
+    Positive angles represent a counter-clockwise rotation.
 
+    Parameters
+    ----------
+    angles : array-like
+        Angles of rotation in deg in the order of axes_order.
+    axes_order : str
+        A sequence of 'x', 'y', 'z' corresponding to axis of rotation.
```

Comment:
Would be useful to give an example of a value since it may not be obvious what is meant is a string like: "xzy" rather than ['x', 'z', 'y'] even though it says it is a string  value.